app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -122,14 +122,32 @@ def response_info
           @response_info ||= create_response_info
         end
 
+        def get_alert_result(log_alert_results, side, alert_name_key, result)
+          if log_alert_results.dig(alert_name_key, side.to_sym).present?
+            alert_value = log_alert_results[alert_name_key][side.to_sym]
+            Rails.logger.
+              info("ALERT ALREADY HAS A VALUE: #{alert_value}, #{result}")
+          end
+          result
+        end
+
         def log_alerts(alerts)
           log_alert_results = {}
+
           alerts.keys.each do |key|
             alerts[key.to_sym].each do |alert|
-              side = alert[:side] || 'no_side'
-              log_alert_results[alert[:name].
+              alert_name_key = alert[:name].
                 downcase.
-                parameterize(separator: '_').to_sym] = { "#{side}": alert[:result] }
+                parameterize(separator: '_').to_sym
+              side = alert[:side] || 'no_side'
+
+              log_alert_results[alert_name_key] =
```

Comment:
This is getting kind of sophisticated and is taking up a lot of room with some duplication in these response objects. Would it make sense to have a `ProcessedAlertToLogAlertResultFormatter` or something like that? Then we could throw a comment into that to describe what this transformation is